apps/openmw/mwworld/projectilemanager.hpp

```diff
@@ -3,6 +3,7 @@
 
 #include <string>
 
+#include <osg/observer_ptr>
```

Comment:
My guess is that the culprit here is FreeBSD's libc++ rather than Clang. When I created the port I tried playing around with different Clang and GCC versions but that lead to a lot of other pain (not related to OpenMW code).

Replacing `const int` with `int` was the most straightforward non-hacky solution to those compiler errors I could find.